.gitignore

```diff
@@ -278,3 +278,4 @@ SDK/**/*.nuspec
 SDK/**/*.snk
 Tests/**/*.snk
 *.snk
+NuGet.config
```

Comment:
It doesn't look required anymore

Reply:
We removed this file, and to avoid random addition this file in the future it should be here.